INTERFACE_SPECIFICATION.md

```diff
@@ -58,7 +58,7 @@ withdraw : (WithdrawArgs) -> (variant { Ok : BlockIndex; Err : WithdrawError });
 ```
 
 This endpoint withdraws the given amount from the caller's account and transfers the cycles to the provided canister ID.
-In addition to the amount and the canister ID (parameter `to`), the caller can provide a subaccount and a `created_at_time` timestamp.
+In addition to the amount and the canister ID (parameter `to`), the caller can provide a subaccount and a `created_at_time` timestamp,. The timestamp is used to deduplicate calls, i.e., a canister is created (at most) once when calling the endpoint multiple times with the same parameters.
```

Comment:
```suggestion
In addition to the amount and the canister ID (parameter `to`), the caller can provide a subaccount and a `created_at_time` timestamp. The timestamp is used to deduplicate calls, i.e., a canister is created (at most) once when calling the endpoint multiple times with the same parameters.
```

Reply:
I messed up the documentation of `created_at_time`, putting things in the wrong place. I now added a brief description for both `withdraw` and `create_canister`. Could you please take another quick look?